network/store/v1/store.go

```diff
@@ -76,6 +76,14 @@ func (s *Store) SaveNetwork(net *network.Network) error {
 	return nil
 }
 
+func (s *Store) DeleteNetwork(name string) error {
+	path := s.GetNetworkPath(name)
+	if err := os.Remove(path); err != nil {
+		return err
+	}
+	return nil
```

Comment:
if you are not wrapping the error, you can :
```suggestion
	return os.Remove(path)
```

Reply:
Thanks, will change